src/extension.ts

```diff
@@ -49,11 +49,13 @@ const DOCUMENT_SELECTOR: DocumentSelector = [
 ];
 
 function initializeExtensionVariables(ctx: vscode.ExtensionContext): void {
+    ext.context = ctx;
+    ext.telemetryOptIn = vscode.workspace.getConfiguration('telemetry').get('enableTelemetry', false) || ext.runningTests;
```

Comment:
I'm going to set `DEBUGTELEMETRY = 1` for our tests instead; otherwise we have to refactor everything around `ext.telemetryOptIn` in order to still be able to test `ActivityMeasurementService`.